### DISCUSSION TITLE
aab Build to SDK Component Error. Help me.

### DISCUSSION THREAD

**Discussion by etudelove** [airsdk/Adobe-Runtime-Support#2049] [Q&A]
aab Build to SDK Component Error. Help me.

**Comment by ajwfrost** [comment]
Hi

This sounds like there's a problem with that Runtime.apk file, not sure if something's gone wrong with your SDK unpacking? Can you check in the path given to ensure the file is there and what the size is (or ideally a checksum like sha256 hash?) - and let us know what version of the SDK you have. But it may just be worth pulling that "air" folder out of the SDK zip file again to ensure the files are okay..

thanks

**Reply by etudelove** [reply]
Adobe AIR SDK version is 33.1.1.929.  It does not work.  Thank You.

**Reply by ajwfrost** [reply]
Hi - it's working for us here. And it wasn't build 929 when you first reported the issue..! so I am wondering if something else is wrong with your installation - are you able to check the file information for that file?
On 33.1.1.929, the details should be:
Size = 13418276
SHA256 checksum = c7b5e8a8072e1b01225224e6784fd50a97242efbf3617a822cf903039f5bdaba

thanks
